components/segment/actions/page/page.mjs

```diff
@@ -0,0 +1,69 @@
+// legacy_hash_id: "a_YEiPdk"
+import { axios } from "@pipedream/platform";
+
+export default {
+  key: "segment-page",
+  name: "Record page views on your website",
+  description: "The page method lets you record page views on your website, along with optional extra information about the page being viewed (note requires userId or anonymousId)",
+  version: "0.2.1",
+  type: "action",
+  props: {
+    segment: {
+      type: "app",
+      app: "segment",
+    },
+    anonymousId: {
+      type: "string",
+      description: "A pseudo-unique substitute for a User ID, for cases when you dont have an absolutely unique identifier. A userId or an anonymousId is required.",
+      optional: true,
+    },
+    context: {
+      type: "object",
+      description: "Dictionary of extra information that provides useful context about a message, but is not directly related to the API call like ip address or locale",
+      optional: true,
+    },
+    integrations: {
+      type: "object",
+      description: "Dictionary of destinations to either enable or disable",
+      optional: true,
+    },
+    name: {
+      type: "string",
+      description: "\tName of the page",
```

Comment:
Should we strip leading / trailing whitespace like this? I'm not sure why it was included in the original description.

Reply:
Leading and trailing whitespace is now stripped from descriptions.